src/qml/QFieldSettings.qml

```diff
@@ -166,8 +174,49 @@ Page {
           anchors.margins: 4
         }
       }
+
+      Item {
+          ColumnLayout {
+              anchors.fill: parent
+              anchors.topMargin: 8
+              anchors.leftMargin: 18
+              anchors.rightMargin: 18
+
+              Label {
+                  text: qsTr( "Select a vertical grid shift in the combo box below to increase vertical location accuracy. Leave blank to disable this feature." )
+                  font: Theme.defaultFont
+
+                  wrapMode: Text.WordWrap
+                  Layout.fillWidth: true
+              }
+
+              ComboBox {
+                  Layout.fillWidth: true
+                  model: [ "" ].concat( platformUtilities.availableGrids() );
```

Comment:
Can we add a text here like "<i>Use raw altitude from positioning device</i>" or similar?

Reply:
@m-kuhn , what about 'No grid (use raw altitude from device)'?